implants/lib/c2/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "c2"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+tonic = { workspace = true }
+prost = "0.12"
+prost-types = { workspace = true }
+tokio = { version = "1.0", features = ["macros", "rt-multi-thread"] }
+
+[build-dependencies]
+tonic-build = "0.10"
```

Comment:
Can we move this to the parent workspace.
If other parts of realm use it in the future would be nice to continue our standard of using one version of a dep across the project.

Reply:
idk how to add build dependencies there, no existing section for them

Reply:
Should be able to add:
```rust
[workspace.build-dependencies]
tonic-build = "0.10"
```
to between dependencies and profile.

Reply:
Doesn't work like that, would require us to add it to the workspace dependencies:

```

Caused by:
  failed to load manifest for dependency `c2`

Caused by:
  failed to parse manifest at `/workspaces/realm/implants/lib/c2/Cargo.toml`

Caused by:
  error inheriting `tonic-build` from workspace root manifest's `workspace.dependencies.tonic-build`

Caused by:
  `dependency.tonic-build` was not found in `workspace.dependencies`
```

Reply:
Added them to the workspace dependencies I guess /shrug